File: CPP/Human.cpp
#include <string>
#include <iostream>

using namespace std;

class Human{
	
	private:
    	string nik;
		string nama;
		string gender;

	public:
		Human(){
            this->nik = "";
            this->nama = "";
            this->gender = "";
		}

		//getter dan setter
        void setNik(string nik){
			this->nik = nik;
		}

		string getNik(){
			return this->nik;
		}

        void setNama(string nama){
			this->nama = nama;
		}

		string getNama(){
			return this->nama;
		}

		void setGender(string gender){
			this->gender = gender;
		}

		string getGender(){
			return this->gender;
		}

		~Human(){
		}
};

File: Python/Program/Human.py
class Human:

	__nama = ''
	__nik = 0
	__gender = ''
	
	def __init__(self, nama = '', nik = 0, gender = ''):
		self.__nama = nama
		self.__nik = nik
		self.__gender = gender
  
	#getter dan setter
	def setNama(self, nama):
		self.__nama = nama

	def getNama(self):
		return self.__nama

	def setNik(self, nik):
		self.__nik = nik

	def getNik(self):
		return self.__nik

	def setGender(self, gender):
		self.__gender = gender

	def getGender(self):
		return self.__gender